examples/permissioned-token/test/failing-paths.js

```diff
@@ -0,0 +1,171 @@
+const { types } = require('@algo-builder/runtime');
+const { assert } = require('chai');
+const {
+  optInToASA,
+  optInToPermissions,
+  issue,
+  whitelist,
+  fund,
+  killToken,
+  transfer,
+  optOut,
+  forceTransfer,
+  setupEnv
+} = require('./common');
+
+describe('Permissioned Token Tests -- Failing Paths', function () {
+  let runtime, master, alice, bob, elon;
+  let lsig, assetIndex, controllerAppID, permissionsAppId;
+
+  // Sync Accounts and ASA def
+  function syncAccounts () {
+    master = runtime.getAccount(master.address);
+    alice = runtime.getAccount(alice.address);
+    bob = runtime.getAccount(bob.address);
+    elon = runtime.getAccount(elon.address);
+  }
+
+  this.beforeEach(async function () {
+    // Create Accounts and Env
+    [
+      runtime,
+      master,
+      alice,
+      bob,
+      elon,
+      assetIndex,
+      controllerAppID,
+      lsig,
+      permissionsAppId
+    ] = setupEnv();
+  });
+
+  it('Token Issuance', () => {
+    // Cannot issue before opting-in
+    assert.throws(() => issue(runtime, alice, elon, 20, controllerAppID, assetIndex, lsig), 'RUNTIME_ERR1404');
+
+    // Opting-in
+    optInToASA(runtime, elon.address, assetIndex);
+    syncAccounts();
+
+    // Cannot issue after killing token
```

Comment:
is it possible to transfer tokens after killing ?

Reply:
please add a test for it.
does this mean balance of each user which has token becomes zero ?

Reply:
or only transfer is prohibited